Client/Extensions/Array+Extension.swift

```diff
@@ -0,0 +1,16 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+extension Array where Element: NSAttributedString {
```

Comment:
We have Array extension in Shared. If this is something we want to be reusable, I would include this inside our existing Array+Extension file there ?